.github/workflows/test-php.yml

```diff
@@ -32,6 +32,11 @@ jobs:
                       woocommerce: '4.7.0'
                       phpunit: '6.5.9'
                       composer: '1.10.19'
+                    - php: '8.0'
```

Comment:
Yeah, it requires a specific `bin` package, I copied this from Woo Core. I agree a separate PR would be better (I thought it would be simple at first).

Reply:
I reverted these changes :)

Reply:
NP Woo Core’s approach for PHP 8 with the `bin` package looks good.